### PR TITLE
Improved docs

### PR DESCRIPTION
**Pull request by amey1355**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Shopware Development Documentation
+# Shopware Development Documentation - Best for Developers
```
Comment (mitelg) [inline README.md:1]: why do want to add this?

Comment (bcremer) [reply README.md:1]: It's most probably hacktoberfest spam.

Comment (mitelg) [reply README.md:1]: I already thought so, too. Will close this

**Comment by mitelg** [comment]
thanks for your contribution @amey1355 :+1: 
but as this brings no value to the documentation, I will close this

**Pull request closed without merging**